mitodl/open-discussions
# Implement mobile widget view

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1518 

#### What's this PR do?
Implements mobile view for widgets. In the mobile view widgets will be removed from the sidebar and shown on a separate page. The user should not see anything different in the desktop view.

#### How should this be manually tested?
Narrow the browser window to mobile width. Browse the channel page and verify that the navigation works as you expect.

#### Any background context you want to provide?
Note that the react renderer will not update after you resize the browser window. You will need to refresh after any resize to test changes.

#### Screenshots (if appropriate)
Desktop:
![desktop](https://user-images.githubusercontent.com/863262/50599593-887ae600-0e7c-11e9-9181-7d6475d518d7.png)

Mobile:
![mobile_channel](https://user-images.githubusercontent.com/863262/50599599-90d32100-0e7c-11e9-92cb-27294b68aed3.png)

![mobile_2](https://user-images.githubusercontent.com/863262/50599603-9466a800-0e7c-11e9-94f1-003257887ce4.png)
